subworkflows/local/locidex.nf

```diff
@@ -1,33 +1,58 @@
 include { LOCIDEX_EXTRACT } from "../../modules/local/locidex_extract.nf"
 include { LOCIDEX_SEARCH } from "../../modules/local/locidex_search.nf"
 include { LOCIDEX_REPORT } from "../../modules/local/locidex_report.nf"
+include { LOCIDEX_SELECT } from "../../modules/local/locidex_select.nf"
 
-
+// TODO add to report fields of `count_loci_found (loci that are not "-")`, `count_loci_missing` (loci that are "-"), total_loci (how many loci screened for), `missing` (total loci - found loci)
```

Comment:
Recommend resolving or creating an issue instead of putting TODOs into PRs.

Reply:
fixed in: https://github.com/phac-nml/mikrokondo/pull/96/commits/0dfe87fbaec3203c9a54f855904edc71b6f2c940